### ISSUE TITLE
mullvad-early-boot-blocking.service fails due to trying to modify the local filesystems before they are read-write

### ISSUE THREAD

**Issue by gco** [mullvad/mullvadvpn-app#4005]
## Issue report

Operating system: Debian 10.13

App version: 2022.5-beta2

### Issue description

The new early-boot systemd service in 2022.5-beta2 fails due to being started before the local filesystems are ready:

```
# journalctl -u mullvad-early-boot-blocking.service
-- Logs begin at Thu 2022-10-06 13:44:34 PDT, end at Thu 2022-10-06 14:10:20 PDT. --
Oct 06 13:44:34 xxx mullvad-daemon[192]: Error: Unable to get log directory
Oct 06 13:44:34 xxx mullvad-daemon[192]: Caused by: Failed to set directory permissions on /var/log/mullvad-vpn
Oct 06 13:44:34 xxx mullvad-daemon[192]: Caused by: Read-only file system (os error 30)
#
```

**Comment by faern** [comment]
Thanks for reporting this! We'll look into how we'll fix it. Maybe have this early service simply not log to file.

**Comment by pinkisemils** [comment]
Should be fixed with #4008.

**Comment by dayvid3** [comment]
I know this was closed, but I'm getting this error with vs. 2022.5 on Fedora 37, DE Cinnamon.

/usr/lib/systemd/system/mullvad-early-boot-blocking.service
Failed to start mullvad-early-boot-blocking.service - Mullvad early boot network blocker.  (systemd)
mullvad-early-boot-blocking.service: Failed with result 'exit-code'.  (systemd)
mullvad-early-boot-blocking.service: Main process exited, code=exited, status=1/FAILURE  (systemd)
Caused by: Read-only file system (os error 30)  (mullvad-daemon)
Caused by: Failed to set directory permissions on /var/log/mullvad-vpn  (mullvad-daemon)
Error: Unable to get log directory

Thanks
![mullvaderror](https://user-images.githubusercontent.com/94722391/202481409-14d53d3e-84ef-40ea-9974-a1e334ebc3ff.png)

**Comment by lunalicca** [comment]
> I know this was closed, but I'm getting this error with vs. 2022.5 on Fedora 37, DE Cinnamon.

I'm seeing the same issue on Fedora 37 with Sway WM. I have /var/log in a BTRFS subvolume, and I can see that the subvolume is being mounted right after the failed attempt from the daemon:

```
Dec 04 12:26:01 artemis mullvad-daemon[982]: Error: Unable to get log directory
Dec 04 12:26:01 artemis mullvad-daemon[982]: Caused by: Failed to create directory /var/log/mullvad-vpn
Dec 04 12:26:01 artemis systemd[1]: Mounting var-log.mount - /var/log...
Dec 04 12:26:01 artemis systemd[1]: Mounted var-log.mount - /var/log.
```

That being said, there are logs going back well into November with the error, before I set up this subvolume layout, so it doesn't appear to be an issue with the subvolume itself. Given that we're both on Fedora, perhaps it's something to do with BTRFS?

**Comment by shoonya75** [comment]
I am experiencing the same issue on Fedora 37, DE KDE.  Mullvad Version 2022.5  This is on ext4 not on BTRFS - nothing has changed on my install other than the mullvad app. 

```
8:59 AM
Failed to start mullvad-early-boot-blocking.service - Mullvad early boot network blocker.
systemd
8:59 AM
mullvad-early-boot-blocking.service: Failed with result 'exit-code'.
systemd
8:59 AM
mullvad-early-boot-blocking.service: Main process exited, code=exited, status=1/FAILURE
systemd
8:59 AM
Caused by: Read-only file system (os error 30)
mullvad-daemon
8:59 AM
Caused by: Failed to set directory permissions on /var/log/mullvad-vpn
mullvad-daemon
8:59 AM
Error: Unable to get log directory
mullvad-daemon
```

**Comment by pinkisemils** [comment]
This is currently fixed in master, as of #4099.
